<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>transact_infow</class>
 <widget class="QWidget" name="transact_infow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Profile</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #bdbdbd;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lbl_transfer_id">
       <property name="minimumSize">
        <size>
         <width>225</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>225</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #111111;</string>
       </property>
       <property name="text">
        <string>Транзакция №</string>
       </property>
       <property name="margin">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_date">
       <property name="minimumSize">
        <size>
         <width>225</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>225</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #111111;</string>
       </property>
       <property name="text">
        <string>Дата отправления</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_sender_id">
       <property name="minimumSize">
        <size>
         <width>225</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>225</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #111111;</string>
       </property>
       <property name="text">
        <string>Id отправителя</string>
       </property>
       <property name="margin">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_sender_name">
       <property name="minimumSize">
        <size>
         <width>225</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>225</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #111111;</string>
       </property>
       <property name="text">
        <string>Имя отправителя</string>
       </property>
       <property name="margin">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_sender_phone">
       <property name="minimumSize">
        <size>
         <width>225</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>225</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #111111;</string>
       </property>
       <property name="text">
        <string>Телефон отправителя</string>
       </property>
       <property name="margin">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_sender_email">
       <property name="minimumSize">
        <size>
         <width>225</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>225</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #111111;</string>
       </property>
       <property name="text">
        <string>e-mail отправителя</string>
       </property>
       <property name="margin">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_value">
       <property name="minimumSize">
        <size>
         <width>225</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>225</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #111111;</string>
       </property>
       <property name="text">
        <string>Сумма</string>
       </property>
       <property name="margin">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_commission">
       <property name="minimumSize">
        <size>
         <width>225</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>225</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #111111;</string>
       </property>
       <property name="text">
        <string>Комиссия</string>
       </property>
       <property name="margin">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_currency">
       <property name="minimumSize">
        <size>
         <width>225</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>225</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #111111;</string>
       </property>
       <property name="text">
        <string>Валюта</string>
       </property>
       <property name="margin">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_msg">
       <property name="minimumSize">
        <size>
         <width>225</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>225</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #111111;</string>
       </property>
       <property name="text">
        <string>Сообщение</string>
       </property>
       <property name="margin">
        <number>6</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="lbl_transfer_id_value">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #111111;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="margin">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_date_value">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #111111;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_sender_id_value">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #111111;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="margin">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_sender_name_value">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #111111;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="margin">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_sender_phone_value">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #111111;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="margin">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_sender_email_value">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #111111;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="margin">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_value_value">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #111111;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="margin">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_commission_value">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #111111;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="margin">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_currency_value">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #111111;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="margin">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_msg_value">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #111111;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="margin">
        <number>6</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
